### ISSUE TITLE
Build workflow failing with missing GLIBC dependency on aarch64 and armv7 slaves

### ISSUE THREAD

**Issue by albinsuresh** [thin-edge/thin-edge.io#1371]
**Describe the bug**

The build workflow is failing on aarch64 and arm-v7 slaves in build workflow with the following error:

```
   Compiling ring v0.16.20
error: failed to run custom build command for `ring v0.16.20`

Caused by:
  process didn't exit successfully: `/target/release/build/ring-a970e8e9fe1552d6/build-script-build` (exit status: 1)
  --- stderr
  /target/release/build/ring-a970e8e9fe1552d6/build-script-build: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.25' not found (required by /target/release/build/ring-a970e8e9fe1552d6/build-script-build)
  /target/release/build/ring-a970e8e9fe1552d6/build-script-build: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.27' not found (required by /target/release/build/ring-a970e8e9fe1552d6/build-script-build)
warning: build failed, waiting for other jobs to finish...
Error: Process completed with exit code 101.
```

Failing build: https://github.com/thin-edge/thin-edge.io/runs/7946483248?check_suite_focus=true

**Comment by initard** [comment]
Some leads or not leads:

- ci pipeline rust toolchain issue? could try changing toolchain version?
- a similar looking issue: https://github.com/cross-rs/cross/issues/724
- or perhaps github updated the containers and there is no old GLIBC version around anymore?

**Comment by rina23q** [comment]
As of August 30, the build workflow and the building part of ci_pipeline started working again. See 
- build workflow https://github.com/thin-edge/thin-edge.io/actions/workflows/build-workflow.yml
- ci_pipeline https://github.com/thin-edge/thin-edge.io/actions/runs/2954040395

However, now installation part of ci_pipeline is failing. Interesting thing is, both on GH hosted runner and self-hosted RPi runners have the same error.

```
+ sudo apt-get --assume-yes install 'mosquitto libmosquitto1 collectd-core'
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package mosquitto libmosquitto1 collectd-core
Error: Process completed with exit code 100.
```

Now we need to focus on this part.

**Comment by rina23q** [comment]
I found the problem. The quote must be removed for `'mosquitto libmosquitto1 collectd-core'`. Now it's considered the one package name. Both [install_for_amd64.sh](https://github.com/thin-edge/thin-edge.io/blob/main/ci/installation_scripts/install_for_amd64.sh) and [install_for_arm.sh](https://github.com/thin-edge/thin-edge.io/blob/main/ci/installation_scripts/install_for_arm.sh) need to be fixed.

This one was changed by [this commit](https://github.com/thin-edge/thin-edge.io/commit/a2c58baddcf685cde414fb66847e6e981921d6c7#diff-3e0d6d69055378f254f6ac487ffdb315c14771e49430f0338207109cd0a420e7L11).

**Comment by PradeepKiruvale** [comment]
Resolved #1375 #1376

**Comment by rina23q** [comment]
Now ci_pileine is green, therefore, close this issue.
